README.md

```diff
@@ -159,18 +164,18 @@ David Tverdota • [GitHub](https://github.com/The-Shader) • [Twitter](https:/
 *Anything to do with the NHS GP flows, which allow someone to get our services
 for free, by making Babylon their NHS GP*
 
-Pedro Loureiro • [GitHub](https://github.com/pedroql)  
 Vasanthakumar Annadurai • [GitHub](https://github.com/apvasanth03)
 
 ---
 
-#### Platform Foundation tribe (Liverpool Street)
+#### Core Alliance tribe (Liverpool Street)
```

Comment:
I think this is still Platform Foundations tribe:
https://babylonpartners.atlassian.net/wiki/spaces/RC/overview